spectests/README.md

```diff
@@ -113,3 +113,36 @@ There are some cases that we decided to skip for now to fasten the time to relea
   - `globals.wast`
 - `SKIP_CALL_INDIRECT_TYPE_MISMATCH`: we implemented traps in a fast way. We haven't covered yet the type mismatch on `call_indirect`. Specs affected:
   - `call_indirect.wast`
+
+- `SKIP_CALL_UNDEFINED_ELEMENT`
+Tables are imported into every spec module, even for modules that don't expect it. We need to figure out a way to prevent import of objects that are not explicitly imported into the module.
+
+Currently cranelift_wasm::ModuleEnvironment does not provide `declare_table_import`, etc. so there is no meaningful way of fixing this yet.
+  - `call_indirect.wast`
+
+- `SKIP_SHARED_TABLE` [elem.wast]
+Currently sharing tables between instances/modules does not work. Below are some of the reasons it is hard to achieve.
```

Comment:
I love seeing this analysis as part of the PR and spectests README.

It will make much easier to dig into a solution in the future and get input/solutions from other people.
Great 👍

Reply:
Thanks